test/test_common/network_utility.cc

```diff
@@ -104,6 +104,11 @@ const std::string addressVersionAsString(const Address::IpVersion version) {
 
 Address::InstanceConstSharedPtr getSomeLoopbackAddress(Address::IpVersion version) {
   if (version == Address::IpVersion::v4) {
+#ifdef __APPLE__
```

Comment:
AFAIK everything is now binding to zero? I would vote to just remove this and treat that case as a test bug if/when it happens.